src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.CURLcode.cs

```diff
@@ -18,6 +18,7 @@ internal enum CURLcode
             CURLE_OPERATION_TIMEDOUT = 28,
             CURLE_ABORTED_BY_CALLBACK = 42,
             CURLE_UNKNOWN_OPTION = 48,
+            CURLE_RECV_ERROR = 56,
```

Comment:
Please add corresponding entries in the native files:
https://github.com/dotnet/corefx/blob/master/src/Native/Unix/System.Net.Http.Native/pal_easy.h#L66
https://github.com/dotnet/corefx/blob/master/src/Native/Unix/System.Net.Http.Native/pal_easy.cpp#L49